### PR TITLE
Introduction

### PR DESCRIPTION
**Pull request by yeland**
Show movie introduction in movie details page. Show all introduction when it is less than or equal to four lines. When the introduction is more than four lines, it will show four lines and read more. And it will show all introduction after clicking the label.

### CODE REVIEW

**Comment by yeland** [comment]
[[E-7]添加影片简介及展开功能 |2|](https://trello.com/c/9vTSBiCd/12-e-7%E6%B7%BB%E5%8A%A0%E5%BD%B1%E7%89%87%E7%AE%80%E4%BB%8B%E5%8F%8A%E5%B1%95%E5%BC%80%E5%8A%9F%E8%83%BD-2)

File: ios-theater/View/IntroduceLabel.swift [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+import UIKit
+
+class IntroduceLabel: UILabel {
+  private var summary = ""
+  
+  func configure(withSummary summary: String) {
+    self.summary = summary
+    let linesArray = summary.getLinesArrayFromLabel(font: self.font, rect: self.frame)
+    var introductionString = NSMutableAttributedString(string: linesArray[0])
```
Comment (zhiying-fan) [inline ios-theater/View/IntroduceLabel.swift]: I think we don't need an initial value for `introductionString`

File: ios-theater/Utils/String+LinesArray.swift [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+//
+//  Created by Linxiao Wei on 2020/1/10.
+//  Copyright © 2020 Linxiao Wei. All rights reserved.
+//
+
+import Foundation
+
+extension String {
+  func getLinesArrayFromLabel(font: UIFont, rect: CGRect) -> [String] {
+    let text:NSString = self as NSString
```
Comment (zhiying-fan) [inline ios-theater/Utils/String+LinesArray.swift]: Code style:
`text: NSString`

Same issues in other places

File: ios-theater/View/IntroduceLabel.swift [hunk 3]
```diff
@@ -14,10 +14,11 @@ class IntroduceLabel: UILabel {
   func configure(withSummary summary: String) {
     self.summary = summary
     let linesArray = summary.getLinesArrayFromLabel(font: self.font, rect: self.frame)
-    var introductionString = NSMutableAttributedString(string: linesArray[0])
+    var introductionString = NSMutableAttributedString(string: "")
```
Comment (zhiying-fan) [inline ios-theater/View/IntroduceLabel.swift:17]: What I mean is `var introductionString: NSMutableAttributedString`

**Review by zhiying-fan — APPROVED** [review]

**Pull request merged**